### ISSUE TITLE
OF Nightly > Release

### ISSUE THREAD

**Issue by dimitre** [openframeworks/openFrameworks#7145]
This is just to inform that OF Nightly has lots of improvement over latest release 0.11.2
You can download on this link https://openframeworks.cc/download/ scrolling to Nightly Builds.

**Comment by dimitre** [comment]
Some highlights of GREAT improvements since 0.11.2. 

c++17 support
Shadows
ARC
Typography metrics fix
AVFoundation player performance fix
XCode improvement (warnings removal, template modernization and project generator)
and a lot more

**Comment by ofTheo** [comment]
yeah be good to roll out a 0.12.0 soon or even a 0.11.3 though at this point I think 0.12.0 would be more accurate.
